lib/street_address/us.rb

```diff
@@ -665,7 +665,15 @@ class << self
       [^\w\x23]*    # skip non-word chars except # (eg unit)
       #{number_regexp} \W*
       (?:#{fraction_regexp}\W*)?
+
+      (
+      # special case for Connecticut (CT)
```

Comment:
Does it make sense to only do it when index is 06xxx ?

Reply:
That could be another way to do this, but I don't really think it's necessary.  The issue really isn't about Connecticut in particular, just about the letters CT being a state and street type.  So it's probably better to keep the check be about the letters CT and not the actual state.